### DISCUSSION TITLE
How to change font size in the title of a plot within quarto dashboard

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#9006] [Q&A] [answered]
### Description

````
```{r}
#| title: Plot
plot(cars)
```
````

**Comment by mcanouil** [comment, marked as answer]
Accessibility request: Please add a proper description in the body of your post.

Also, please share a small **self-contained "working" (reproducible)** example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

This being said, Dashboards are HTML, so you can use your favourite browser and use its inspect/developer mode to target the proper HTML/CSS selector, then create a CSS rule you can include via `css` or `include-` YAML keys.
This is demonstrated in many [discussions](https://github.com/quarto-dev/quarto-cli/discussions?discussions_q=is%3Aopen+label%3Ahtml+label%3Athemes+inspect). 

---

You can share a **self-contained "working" (reproducible)** Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).
See <https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines>.

If you have multiple files (and if it is **absolutely required** to have multiple files), please share as a Git repository.

<table>
<tr><th>R</th><th>Python</th></tr>
<tr><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: knitr
---

This is a reproducible Quarto document.

```{r}
x <- c(1, 2, 3, 4, 5)
y <- c(1, 4, 9, 16, 25)

plot(x, y)
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td><td>

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
engine: jupyter
---

This is a reproducible Quarto document.

```{python}
import matplotlib.pyplot as plt

x = [1, 2, 3, 4, 5]
y = [1, 4, 9, 16, 25]

plt.plot(x, y)
plt.show()
```

![An image](https://placehold.co/600x400.png)

The end.
````
`````

</td></tr>
</table>

Additionally and if not already given, please share the output of `quarto check` within a code blocks (*i.e.*, using three backticks ` ```txt `), see <https://quarto.org/bug-reports.html#check>.

**Comment by ManuelSpinola** [comment]
Thank you very much.

Manuel

El jue, 7 mar 2024 a las 15:16, Mickaël Canouil ***@***.***>)
escribió:

> Accessibility request: Please add a proper description in the body of your
> post.
>
> Also, please share a small *self-contained "working" (reproducible)*
> example to work with, *i.e.*, a complete Quarto document or a Git
> repository? Thanks.
>
> This being said, Dashboards are HTML, so you can use your favourite
> browser and use its inspect/developer mode to target the proper HTML/CSS
> selector, then create a CSS rule you can include via css or include- YAML
> keys.
> This is demonstrated in many discussions
> <https://github.com/quarto-dev/quarto-cli/discussions?discussions_q=is%3Aopen+label%3Ahtml+label%3Athemes+inspect>
> .
> ------------------------------
>
> You can share a *self-contained "working" (reproducible)* Quarto document
> using the following syntax, *i.e.*, using more backticks than you have in
> your document (usually four ````).
> See
> https://quarto.org/bug-reports.html#small-is-beautiful-aim-for-a-single-document-with-10-lines
> .
>
> If you have multiple files (and if it is *absolutely required* to have
> multiple files), please share as a Git repository.
> RPython
>
> ````qmd
> ---
> title: "Reproducible Quarto Document"
> format: html
> engine: knitr
> ---
>
> This is a reproducible Quarto document.
>
> ```{r}
> x <- c(1, 2, 3, 4, 5)
> y <- c(1, 4, 9, 16, 25)
>
> plot(x, y)
> ```
>
> ![An image](https://placehold.co/600x400.png)
>
> The end.````
>
> ````qmd
> ---
> title: "Reproducible Quarto Document"
> format: html
> engine: jupyter
> ---
>
> This is a reproducible Quarto document.
>
> ```{python}
> import matplotlib.pyplot as plt
>
> x = [1, 2, 3, 4, 5]
> y = [1, 4, 9, 16, 25]
>
> plt.plot(x, y)
> plt.show()
> ```
>
> ![An image](https://placehold.co/600x400.png)
>
> The end.````
>
> Additionally and if not already given, please share the output of quarto
> check within a code blocks (*i.e.*, using three backticks ```txt), see
> https://quarto.org/bug-reports.html#check.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/9006#discussioncomment-8712649>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFI3FBZ27V6L3QPZFKUWKIDYXDKJZAVCNFSM6AAAAABELWPR3WVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DOMJSGY2DS>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Manuel Spínola, Ph.D.*
Instituto Internacional en Conservación y Manejo de Vida Silvestre
Universidad Nacional
Apartado 1350-3000
Heredia
COSTA RICA
***@***.*** ***@***.***>
***@***.***
Teléfono: (506) 8706 - 4662
Sitio web institucional: ICOMVIS
<http://www.icomvis.una.ac.cr/index.php/manuel>
Sitio web personal: Sitio personal <https://mspinola-sitioweb.netlify.app>
Blog sobre Ciencia de Datos: Blog de Ciencia de Datos
<https://mspinola-ciencia-de-datos.netlify.app>
